assets/src/components/detours/diversionPanel.tsx

```diff
@@ -67,7 +68,13 @@ export const DiversionPanel = ({
       {missedStops && (
         <section className="pb-3">
           <h2 className="c-diversion-panel__h3">Missed Stops</h2>
-          {missedStops}
+          <ListGroup as="ol">
+            {missedStops.map((missedStop) => (
+              <ListGroup.Item key={missedStop.id}>
```

Comment:
issue: Are these ID's unique?

Reply:
But a stop could be visited twice, so that would mean these are not unique, correct?

Reply:
We don't have stop sequence in the data we get back, but I think the correct way to handle this is actually to just remove duplicated stops from the list, since the list is basically "which stops on this route are somehow affected by the detour."

Reply:
I disagree, In my mind, this UI element represents the ordered list of stops that are missed. So to me, that means a stop that is missed twice would appear twice.

But I think we're both making assumptions about what should be here, so maybe we should ask the larger team?

If we do the "which stops on this route are somehow affected by the detour." version, I think we should switch this `<ListGroup>` to a `ul`

Reply:
The purpose of this data is to be used in alerts, and alerts can only select stop IDs, not stop IDs combined with stop sequences: https://gtfs.org/realtime/reference/#message-entityselector

That said I agree that if this is the case we should change to a `ul`. I'll do that along with the other updates and we can ask the larger group, but I don't want to block this PR on that.

Reply:
I still have to actually make those updates, but one interesting thing is that on the back-end we rely on `shape_with_stops_for_trip`, which in turn calls `stops_for_trip`, which in turn doesn't do anything special to handle stops that are visited twice on the same trip. So just flagging that if we do want to be sensitive to stop sequence here, we would need to revisit some of that logic as well.